kvm.go

```diff
@@ -259,7 +274,11 @@ func (d *Driver) validatePrivateNetwork() error {
 		"255.255.255.0",
 		"192.168.42.2",
 		"192.168.42.254")
-	network, err = d.conn.NetworkDefineXML(xml)
+
+	if err != nil {
+		return err
+	}
```

Comment:
This is additional error check somehow got in.  The network creation (NetworkDefineXML(xml)) is below (line 281) with an appropriate error catch.  I will remove this useless error catch.